src/clojure_tictactoe/game/players.clj

```diff
@@ -0,0 +1,11 @@
+(ns clojure-tictactoe.game.players)
+
+(def player-atom (atom "x"))
+
+(defn switch-player [current-player]
+  (if (= current-player "x")
+    "o"
+    "x"))
+
+(defn player-swap []
+  (swap! player-atom switch-player))
```

Comment:
Have you thought about how to switch between players without using atoms?

Reply:
I built that into my estimate for user-can-choose-token.

Reply:
👍